morpheushelper/cogs/rules.py

```diff
@@ -1,9 +1,9 @@
 from http.client import HTTPException
-from typing import Optional
+from typing import Optional, Tuple, List
```

Comment:
am i wrong or is it possible to delete this?

Reply:
you are totally right, I forgot about this